starboard/xb1/shared/gpu_base_video_decoder.cc

```diff
@@ -54,7 +54,7 @@ using Windows::Graphics::Display::Core::HdmiDisplayInformation;
 // Limit the number of pending buffers.
 constexpr int kMaxNumberOfPendingBuffers = 8;
 // Limit the cached presenting images.
-constexpr int kNumberOfCachedPresentingImage = 2;
+constexpr int kNumberOfCachedPresentingImage = 3;

```

Comment:
Should we release memory somewhere else to compensate this additional image?

Reply:
Nope, this constant is not related to image allocation (or allocation in general). It only determines for how long to hold image before it can be reused.